test/data/validation/destinations/ga4.json

```
@@ -317,6 +317,10 @@
           {
             "provider": "ketch",
             "consents": []
+          },
+          {
+            "provider": "iubenda",
+            "consents": ["1", "4"]
```

The format is incorrect. See the above "custom" provider values for reference.

```suggestion
            "consents": [
              {
                "consent": "1"
              },
              {
                "consent": "4"
              }
            ]
```